scripts/get-workspace-map.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# get-workspace-map.sh
+#   Read Namespace resources and generate a JSON object mapping from namespaces
+#   to workspace based on the value of label toolchain.dev.openshift.com/space.
+#   This script assumes `oc` is preconfigured with all the required clusters
+#   and could be found in $PATH.
+#
+#   if the CONTEXTS environment variable is set, the script will query
+#   locally-defined contexts with those names. Otherwise, the script will query
+#   contexts associated with a hard-coded list of staging clusters.
+#
+set -o pipefail -o errexit -o nounset
+
+# Space-separated locally-defined context names to be queried
+CONTEXTS=(${CONTEXTS:-""})
+stage_clusters=$(cat <<- EOM
```

Comment:
since this is used only in the conditional statement, any reason not to set it there?

Reply:
Yes. I wanted it to be seen as a constant. Suppose that tomorrow the list of staging clusters changes, I want this constant to be at the top of the file so it's easy to locate and update.